language-configuration.json

```diff
@@ -15,8 +15,8 @@
     ],
     // symbols that are auto closed when typing
     "autoClosingPairs": [
-        ["{{", "}}"],
         ["{", "}"],
+        ["{{-", "-"],
```

Comment:
Again, this is just my assumption, why not just add `["-", "-"]` instead of `["{{-", "-"]`?

Reply:
we don't want to have `--` for every `-` user types. do we?

Reply:
Ah I see